handbook/product/design/index.md

```diff
@@ -1,132 +1,82 @@
-# Product Design
+# Product design
 
-<img width="60%" height="60%" src="design-team-logo.jpg" alt="Sourcegraph Design team logo">
+### Welcome to product design at Sourcegraph!
 
-The role of the product design team is to make validated decisions that enhance the user experience of Sourcegraph.
+The product design team exists to increase the value Sourcegraph delivers to its users.  We achieve this objective through thoughtful discovery, inspired design, careful validation, and close collaboration with all manner of roles throughout the company. 
 
-Here are a few of the things we strive to make true:
+We also seek to make it easier and more efficient to build Sourcegraph itself. Through the artifacts of our open process, we provide a forum where anyone in the company can provide valuable input that improves a project's outcome before a single line of code is written.
 
-- Advocate for our users by collaborating with them
-- Perform our due diligence when making design decisions, by researching and testing solutions within scope
-- Proactive, with a bias towards action
-- Communicate openly, and solicit feedback frequently
-- To be egoless when soliciting responses to our work
+## Team
 
-## Contact
+- [Alicja Suska](../../company/team/index.md#alicja-suska-she-her) (she/her)
+- [Rob Rhyne](../../company/team/index.md#rob-rhyne) (he/him)
+- [Quinn Keast](../../company/team/index.md#quinn-keast-he-him) (he/him) – [ReadMe](https://quinnkeast.com/readme)
+- S.L. (joining in August)
 
-- #design channel or @design-team on Slack
 
-## Working with design / requesting design work
+<img width="30%" height="30%" src="design-team-logo.jpg" alt="Sourcegraph Design team logo">
 
-Working with design should feel fluid, you should be able to open an issue or bring up a concern in Slack, and receive actionable feedback: either implementable, or a call to explore further.
+We’re hiring! [Check out our open roles](https://boards.greenhouse.io/sourcegraph91).
 
-You should also not feel limited to words, images (sketches, or references made by editing the browser DOM) are also encouraged. 
+## Defining success
```

Comment:
This section feels less like we are defining success and more like we are describing our team ethos. 

How about **Design team ethos** or **Guiding principles** or similar?

Reply:
```suggestion
## Design team ethos
```

Design team ethos makes sense to me!